src/calendar/view/CalendarAgendaView.ts

```diff
@@ -34,9 +36,14 @@ export type CalendarAgendaViewAttrs = {
 	/**  when the selected date was changed  */
 	onDateSelected: (date: Date) => unknown
 	eventPreviewModel: CalendarEventPreviewViewModel | null
+	viewModel: CalendarAgendaViewModel
+	scheduler: LazyLoaded<Scheduler>
```

Comment:
I think lazy scheduler makes things too complicated. Scheduler is mostly needed as a injectable abstraction in other places, here I would simply do it like in `MultiDayCalendarView` with `setInterval`. We could even just move it from `MultiDayCalendarView` to `CalendarView`.